pkg/graph/schema.resolvers.go

```diff
@@ -43,18 +43,13 @@ func (r *accessibilityRequestResolver) Documents(ctx context.Context, obj *model
 }
 
 func (r *accessibilityRequestResolver) RelevantTestDate(ctx context.Context, obj *models.AccessibilityRequest) (*models.TestDate, error) {
-	allDates := []*models.TestDate{}
-
-	if time.Now().Unix()%3 == 0 {
-		allDates = append(allDates, &models.TestDate{
-			ID:       uuid.New(),
-			Date:     time.Now().AddDate(0, 0, 1),
-			TestType: models.TestDateTestTypeInitial,
-		})
+	allDates, err := r.store.FetchTestDatesByRequestID(ctx, obj.ID)
+	if err != nil {
+		return nil, err
 	}
+
 	var nearFuture *models.TestDate
```

Comment:
I might rename these `nextUpcoming` and `mostRecent` for clarity

Reply:
Naming things is one of the hardest things in software engineering! I'll skip the rename for this moment to get the PR in quickly, but open to talking through these names.